pkg/backup/backup.go

```diff
@@ -1107,3 +1121,21 @@ func putVolumeInfos(
 
 	return backupStore.PutBackupVolumeInfos(backupName, backupVolumeInfoBuf)
 }
+
+func getNamespacesManagedByArgoCD(kbClient kbclient.Client, includedNamespaces []string, log logrus.FieldLogger) []string {
+	var nsManagedByArgoCD []string
+
+	for _, nsName := range includedNamespaces {
+		ns := corev1api.Namespace{}
+		if err := kbClient.Get(context.Background(), kbclient.ObjectKey{Name: nsName}, &ns); err != nil {
+			log.WithError(err).Errorf("error getting namespace %s", nsName)
+			continue
+		}
+
+		nsLabels := ns.GetLabels()
+		if len(nsLabels[ArgoCDManagedByNamespaceLabel]) > 0 {
```

Comment:
Should we check if label exist here instead of checking if its value is not an empty string?

Reply:
That would probably be better.. as long as we know it is possible label only need key and empty value is ok

Reply:
The rationale here is the having an empty value for the ArgoCD label key would not make much sense, so this acts as a additional check.

Reply:
While it may not make sense, we need to match whatever argocd logic is regarding namespace it manages. If it disallows empty value label then current code is fine.

Reply:
Looks like the value here should never be empty according to [docs](https://github.com/argoproj-labs/argocd-operator/blob/master/docs/usage/deploy-to-different-namespaces.md). Value would be namespace where argocd instance lives. An Argo CD instance always live in a namespace, and namespaces are never empty string.

We can just check if length is > 0 in this case.